src/api-docs/source/includes/_wallet_rpc.md

```diff
@@ -156,19 +156,16 @@ const walletClient = new WalletClient(walletOptions);
 
 ```json
 {
-  "result": {
-    "walletid": "primary",
-    "walletversion": 6,
-    "balance": 50,
-    "unconfirmed_balance": 50,
-    "txcount": 1,
-    "keypoololdest": 0,
-    "keypoolsize": 0,
-    "unlocked_until": 0,
-    "paytxfee": 0
-  },
-  "error": null,
-  "id": null
```

Comment:
These metadata will still be present in an RPC call

Reply:
@pinheadmz updated.